src/app/shared/store/meta-data.state.ts

```
@@ -364,4 +382,12 @@
   clearCodeficatorSearch({ patchState }: StateContext<MetaDataStateModel>, {}: ClearCodeficatorSearch): void {
     patchState({ codeficatorSearch: [] });
   }
+
+  @Action(GetLanguageList)
+  getLanguageList({ patchState }: StateContext<MetaDataStateModel>, {}: GetLanguageList): Observable<LanguageListItem[]> {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)